source/configure/integrations-configuration-settings.rst

```diff
@@ -143,8 +159,8 @@ Enable integrations to override usernames
   :configjson: .ServiceSettings.EnablePostIconOverride
   :environment: MM_SERVICESETTINGS_ENABLEPOSTICONOVERRIDE
 
-  - **True**: Webhooks, slash commands, and other integrations, such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`__, will be allowed to change the profile picture they post with.
-  - **False**: Webhooks, slash commands, and OAuth 2.0 apps can only post with the profile picture of the account they were set up with.
+  - **true**: Webhooks, slash commands, OAuth 2.0 apps, and other integrations such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`__, will be allowed to change the username they are posting as.
+  - **false**: **(Default)** Custom slash commands can only post as the username of the user who used the slash command.
```

Comment:
That is not true. This settings is about the profile picture.

Reply:
Thanks! I fixed this too in my latest commit.